backend/routers/routes/posts.py

```diff
@@ -0,0 +1,55 @@
+from typing import Annotated, Literal
+
+from crud import post_crud, user_crud
+from fastapi import APIRouter, Depends, HTTPException, Query, Request, Response
+from models.post_models import FilterParams, Post, PostEdit
+
+post_router = APIRouter()
+
+
+@post_router.post("")
```

Comment:
for the endpoints make a response model:
`@document_router.post("", response_model=DocumentResponse)`